integration-cli/environment/clean.go

```diff
@@ -53,7 +53,15 @@ func getPausedContainers(t testingT, dockerBinary string) []string {
 func deleteAllContainers(t testingT, dockerBinary string) {
 	containers := getAllContainers(t, dockerBinary)
 	if len(containers) > 0 {
-		icmd.RunCommand(dockerBinary, append([]string{"rm", "-fv"}, containers...)...).Assert(t, icmd.Success)
+		result := icmd.RunCommand(dockerBinary, append([]string{"rm", "-fv"}, containers...)...)
+		if result.Error != nil {
+			// If the error is "No such container: ..." this means the container doesn't exists anymore,
+			// we can safely ignore that one.
+			if strings.Contains(result.Stderr(), "No such container") {
+				return
```

Comment:
> Also the z CI is failing because of " removal of container CID is already in progress", should that also be handled here?

Oh, I should take a look at that build too. The removal is already in progress means that we won't be able to remove it *client-side* (i.e. you're sure to get "no such container" later). `rm` will do all the remove and then report all errors. But yeah we need to improve that somehow 😓